### ISSUE TITLE
[Feature request] Whitelabel

### ISSUE THREAD

**Issue by AndersJpg** [weDevsOfficial/happy-elementor-addons#29]
It's pretty simple would love to get Whitelabel. Special for us agency :)

**Comment by n10city** [comment]
Ditto! What they said. ^^^

**Comment by AndersJpg** [comment]
+999 for this

**Comment by heyfletch** [comment]
Yes, please provide ways to at least start to remove the over abundance of "Happy" inserted into our admin panels. Our clients are overwhelmed by all this third party stuff thrown in there... they don't care if it's Happy, Ultimate Addons for Elementor, etc. UAE, for example, provides a white label option. 

It also can be confusing. Why is a "clone" feature, "Happy Clone"? I mean, you really have to think about whether the cloning does something in a different way than just cloning. No one cares... "Oh, this function is provided by Happy Addons". Just call it Clone like other Clone plugins do. 

Also, I see there's now a magic button to insert Happy Templates. I don't want to use that feature, or at the very least, it should be integrated into the default Elementor Template library (like many other Elementor Addon plugins do). Every time I go to insert a new section, the button stares at me making me wonder which of the 3 buttons I need to choose to get a blank section or templates.

Also, I now see that if I activate Happy Addons (free) and do not activate my Pro license, Happy Addons jacks up its widget section to the top of my Elementor Widgets area, promoting the Pro widgets. That is SUPER annoying - you are defacing that area and making it more difficult for people to use Elementor to get their job done. At least provide a way to turn that off or move it to the bottom... especially for people like me that actually have a Pro license but do not want to activate it on sites that don't need it. 

So, please provide a "White Label" option OR the ability to remove some of the Happy Invasion of our admin. I know you like it for brand recognition purposes, but it has no place in our admin panels as us or our clients are trying to get to work done. It actually makes me want to use and promote Happy Addons LESS. 

Note, I requested a part of this back in Dec 2019, and here was the response, so now that multiple clients are asking for it, I do hope you implement. Thanks!

![image](https://user-images.githubusercontent.com/658245/87787738-1f628580-c80a-11ea-9ed7-f992608eed48.png)

**Comment by AndersJpg** [comment]
Any status on this

**Comment by AndersJpg** [comment]
> Yes, please provide ways to at least start to remove the over abundance of "Happy" inserted into our admin panels. Our clients are overwhelmed by all this third party stuff thrown in there... they don't care if it's Happy, Ultimate Addons for Elementor, etc. UAE, for example, provides a white label option.
> 
> It also can be confusing. Why is a "clone" feature, "Happy Clone"? I mean, you really have to think about whether the cloning does something in a different way than just cloning. No one cares... "Oh, this function is provided by Happy Addons". Just call it Clone like other Clone plugins do.
> 
> Also, I see there's now a magic button to insert Happy Templates. I don't want to use that feature, or at the very least, it should be integrated into the default Elementor Template library (like many other Elementor Addon plugins do). Every time I go to insert a new section, the button stares at me making me wonder which of the 3 buttons I need to choose to get a blank section or templates.
> 
> Also, I now see that if I activate Happy Addons (free) and do not activate my Pro license, Happy Addons jacks up its widget section to the top of my Elementor Widgets area, promoting the Pro widgets. That is SUPER annoying - you are defacing that area and making it more difficult for people to use Elementor to get their job done. At least provide a way to turn that off or move it to the bottom... especially for people like me that actually have a Pro license but do not want to activate it on sites that don't need it.
> 
> So, please provide a "White Label" option OR the ability to remove some of the Happy Invasion of our admin. I know you like it for brand recognition purposes, but it has no place in our admin panels as us or our clients are trying to get to work done. It actually makes me want to use and promote Happy Addons LESS.
> 
> Note, I requested a part of this back in Dec 2019, and here was the response, so now that multiple clients are asking for it, I do hope you implement. Thanks!
> 
> ![image](https://user-images.githubusercontent.com/658245/88400938-10924a80-cd97-11ea-9978-e58a3a5927ee.png)

Really great feedback. 

Any news on all this Whitelabel dear happy team ;)
